proto/prehog/v1alpha/teleport.proto

```diff
@@ -1556,6 +1556,12 @@ message UserTaskStateEvent {
   int32 instances_count = 4;
 }
 
+// AccessRequestEvent emitted for Access Request audit events.
+message AccessListReviewEvent {
+  // Teleport user name. Anonymized.
+  string user_name = 1;
```

Comment:
Don't we need implement the 
```
AccessListReviewEvent) Anonymize(a utils.Anonymizer) prehogv1a.SubmitEventReques
```

method to anonymize the user name ?

Reply:
Oh my bad! This was used in earlier commit by I changed that in https://github.com/gravitational/teleport/pull/52183/commits/dec9c23962d07d89bc7833c5cc93d84e8ad8a2ef but missed to delete this one.

Fixed in https://github.com/gravitational/teleport/pull/52183/commits/72aa6928c010ba2ec42de96ad9ecae0d526636e6 by removing the proto definition